packages/lib-react-components/src/ZooFooter/components/LinkList/LinkList.js

```diff
@@ -0,0 +1,46 @@
+import { Anchor, Box } from 'grommet'
+import { arrayOf, string} from 'prop-types'
+import React from 'react'
+import { zipWith } from 'lodash'
+import zooTheme from '@zooniverse/grommet-theme'
+
+import zipLabelsAndUrls from '../../helpers/zipLabelsAndUrls'
+import SpacedText from '../../../SpacedText'
+
+export default function LinkList ({ colorTheme, labels, urls }) {
+  const [title, ...links] = zipLabelsAndUrls(labels, urls)
+  const colors = zooTheme[colorTheme].zooFooter.linkList
+
+  return (
+    <Box direction='column' gap='xxsmall' tag='nav'>
+
+      <SpacedText weight='bold'>
+        <Anchor href={title.url} color={colors.title}>
+          {title.label}
+        </Anchor>
+      </SpacedText>
+
+      {links.length > 0 && links.map(link => (
+        <Anchor
+          color={colors.item}
+          href={link.url}
+          key={link.url}
+          size='small'
```

Comment:
Using Zooniverse Classrooms as an inspectable reference (https://classroom.zooniverse.org/), I think these links should be a smaller font size. I think `'xsmall'` would work here.

Reply:
It's a toss-up; in that link you provide, they're rounded to `13px`; `xsmall` is `12px`, and `small` is `14px`. I'm going to err on the side of legibility and keep them at `small`.

Reply:
ok, fair enough.